velox/connectors/Connector.h

```diff
@@ -171,20 +179,30 @@ class ConnectorQueryCtx {
  public:
   ConnectorQueryCtx(
       memory::MemoryPool* pool,
-      Config* config,
+      const core::QueryConfig& queryConfig,
```

Comment:
Why do we need to pass both queryConfig and connectorConfig? Why is one passed as const & and the other as raw pointer?

Reply:
Mash, do you know if we can derive one of queryConfig and connectorConfig from another? I traced down to https://fburl.com/mx1hb3yx and figured they might be different.

Will change QueryConfig to raw pointer

Reply:
Config is expected to provide engine-specific properties, while ConnectorConfig provides connector-specific properties. The connectors are expected to use ConnectorConfig only. Why do you need to pass Config? Which properties does the connector need? Can we pass these explicitly?